datalake-api/src/main/java/com/sequenceiq/sdx/api/model/SdxClusterDetailResponse.java

```diff
@@ -20,7 +20,7 @@ public SdxClusterDetailResponse(SdxClusterResponse sdxClusterResponse, StackV4Re
                 sdxClusterResponse.getCloudStorageFileSystemType(), sdxClusterResponse.getRuntime(),
                 sdxClusterResponse.getRangerRazEnabled(), sdxClusterResponse.getTags(), sdxClusterResponse.getCertExpirationState(),
                 sdxClusterResponse.getSdxClusterServiceVersion(), sdxClusterResponse.isDetached(),
-                sdxClusterResponse.isEnableMultiAz());
+                sdxClusterResponse.isEnableMultiAz(), sdxClusterResponse.getDatabaseEngineVersion());
```

Comment:
Nit: This is the 18th argument in this constructor variant. Perhaps we shall spend some time to refactor this with a builder. :smile:

Reply:
+1